surelock/src/main/java/com/smashingboxes/surelock/Surelock.java

```diff
@@ -188,17 +200,21 @@ public void store(String key, byte[] value) {
     /**
      * Login with the default Surelock dialog fragment
      *
-     * @param key pointer in storage to encrypted value
-     * @param fragmentManager
-     * @param fingerprintDialogFragmentTag
+     * @param key                          pointer in storage to encrypted value
+     * @param fragmentManager              The FragmentManger used to load the dialog fragment
+     * @param fingerprintDialogFragmentTag The tag associated with the dialog fragment
+     * @param styleId                      The resource id of the style set for custom dialog
+     *                                     attributes
      */
     public void loginWithFingerprint(@NonNull String key,
                                      @NonNull FragmentManager fragmentManager,
-                                     String fingerprintDialogFragmentTag) {
+                                     String fingerprintDialogFragmentTag,
+                                     @StyleRes int styleId) {
         SurelockDefaultDialog fragment = (SurelockDefaultDialog) fragmentManager.findFragmentByTag(fingerprintDialogFragmentTag);
         if (fragment == null) {
             fragment = (SurelockDefaultDialog) SurelockDefaultDialog.instantiate(context, SurelockDefaultDialog.class.getName());
         }
+        fragment.setStyleId(styleId);
```

Comment:
Instead of adding a new method here, could we just create a `newInstance()` method in our SurelockDefaultDialog class which takes in styleId as the only parameter? This way, we'll be future-proofed in case we need to add more parameters.